src/test/resources/pacts/publicapi-direct-debit-connector-get-events.json

```diff
@@ -38,8 +38,8 @@
         "method": "GET",
         "path": "/v1/events",
         "query": {
-          "to_date": ["2018-03-13T10:00:05Z"],
-          "from_date": ["2018-03-13T10:00:03Z"],
+          "to_date": ["2018-03-13T10:00:05.000Z"],
+          "from_date": ["2018-03-13T10:00:03.000Z"],
```

Comment:
Again, not sure about changing these

Reply:
This change is required as we create this request inside `publicapi` and we call `connector` with it. Milliseconds are added in `DateTimeUtils#toUTCDateString` as we use this method everywhere to format ZonedDateTime. We do not receive this parameters from outside world :)

Reply:
Ah, OK, I misunderstood where it was coming from